src/Admin/BaseFieldDescription.php

```diff
@@ -460,6 +460,16 @@ public function getHelp()
         return $this->help;
     }
 
+    public function getHelpTranslationParameters()
+    {
+        return $this->getOption('help_translation_parameters');
+    }
+
+    public function getLabelTranslationParameters()
+    {
+        return $this->getOption('label_translation_parameters');
+    }
```

Comment:
I'm not sure we want to have quick getter for every option.

Shouldn't we remove them and only use 
`options. label_translation_parameters` to access this kind of property ?

Reply:
ok, just misunderstood your comment, i thought you want it accessible via getter

> > > 
> So if I want to get `foo`, how can I know if I need to write
> 
> * `foo`
> * `sonata_admin.field_description.foo`
> * `sonata_admin.field_description.options.foo`
>   ?

Reply:
I never correctly understood the way this file works.

What I wanted to say is that I don't know why sometimes we're using `foo` and sometimes we're using `sonata_admin.field_description.options.foo`. Which one should we use to this work properly ? And how to know this ?

We're indeed also using `sonata_admin.field_description.foo` sometimes but I feel like it's not a good idea to always add getter, because we need to add them to the interface (and it's a BC-break). Working with $options is easier.

In a PR I prepare for 5.0 (so in a long long time), I will try to remove some `sonata_admin.field_description.foo` getter.
https://github.com/sonata-project/SonataDatagridBundle/pull/239